engines/query-sparql-solid/package.json

```diff
@@ -44,139 +44,21 @@
     "engine-default.js"
   ],
   "dependencies": {
-    "@comunica/actor-context-preprocess-source-to-destination": "^2.5.1",
```

Comment:
These dependencies should be generated using https://github.com/comunica/comunica/tree/master/packages/packager